src/nuget/packages.config

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="xunit" version="2.0.0-beta5-build2785" targetFramework="net45" />
-  <package id="xunit.abstractions" version="2.0.0-beta5-build2785" targetFramework="net45" />
+  <package id="xunit.abstractions.netcore" version="1.0.0-prerelease" targetFramework="net45" />
```

Comment:
I'm fine with updating this one as well but I actually don't think this file is used by anyone, are you aware of any usages? It is on my list for removing when I tackle #621.